generator/templates/Authorisation/src/api/endpoints/password/reset.js

```diff
@@ -2,9 +2,9 @@ import { post } from '../../implementation/app';
 
 export default async function (email, token, password, passwordConfirmation) {
   return post('registration', {
```

Comment:
```suggestion
export default async (email, token, password, passwordConfirmation) => post('registration', { email, token, password, passwordConfirmation });
```

Volgens mij werkt dit ook :smile_cat:

Reply:
Moet de rest wel weg. Ik kon niet alles selecteren omdat er verwijderde regels in zatten